nixos/modules/programs/chromium.nix

```
@@ -122,7 +122,7 @@ in
   ###### implementation
 
   config = lib.mkIf cfg.enable {
-    environment.systemPackages = [ cfg.package ];
+    environment.systemPackages = lib.optional (cfg.package != null) cfg.package;
```

```suggestion
    environment.systemPackages = lib.optionals (cfg.package != null) [
      cfg.package
    ];
```

---

Besides this seeming to be common among other modules allowing a nullable package option, I see no reason to use optionals here?

---

This is an opinionated suggestion, this is the reason why it is just a comment.

I suggested to use `optionals` (_plural form, it's important_) here for two reasons:
1) Not the most important... but `environment.systemPackages` is plural form and semantically it makes more sense. Also, it requires a list, and it's perhaps better to use `lib.optionals` because of reason 2 hereunder.
2) The brackets needs to be added explicitly, lowering the barrier and cognitive load of the Nix language

---

Wouldn't the nixfmt-rfc-style be

```suggestion
    environment.systemPackages = lib.optionals (cfg.package != null) [ cfg.package ];
```

all one line?

---

Yes most probably but, the style is not the point here.

---

> Yes most probably but, the style is not the point here.

I know it wasn't. Just something I felt the need to point out.

---

We don't have a `lib.optional` v. `lib.optionals` policy/preference in nixpkgs, both are used with massive frequency, and even semantically I don't see us installing more than one package in this module. Can we drop this comment?